v2/uci/src/modules/user-segment/fusionauth/fusionauth.service.spec.ts

```diff
@@ -4,19 +4,11 @@ import FusionAuthClient, { ApplicationRequest, RegistrationRequest, UUID } from
 import { ConfigService } from '@nestjs/config';
 import { FusionAuthClientProvider } from './fusionauthClientProvider';
 
-
 class MockFusionAuthClient {
   callCounter = 0;
 
   async createApplication(_botID: string, _request: ApplicationRequest): Promise<any> {
-    if (this.callCounter == 0) {
-      this.callCounter = 1;
-      return Promise.reject('Test reject');
-    }
-    else {
-      this.callCounter = 0;
```

Comment:
Please revert this change, this logic was put in place for `FusionAuthClient` retries, or find a workaround logic for this.